### PR TITLE
logging-fluentd ConfigMap is overwritten during logging stack upgrades

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1875191

### CODE REVIEW

**Comment by mburke5678** [comment]
@ewolinetz PTAL

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -783,6 +783,11 @@ upgrade the component.
 To upgrade an existing EFK logging stack deployment, you review your parameters
 and run the *_openshift-logging/config.yml_* playbook.
 
+[NOTE]
+====
+The upgrade can override your `logging-fluentd` and `logging-curator` ConfigMaps. If you want to retain your current `logging-fluentd` and `logging-curator` ConfigMaps, set the `openshift_logging_fluentd_replace_configmap` and `openshift_logging_curator_replace_configmap` parameters to `no` in the the xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory host file].
```
Comment (ewolinetz) [inline upgrading/automated_upgrades.adoc:788]: s/override/replace/

**Comment by mburke5678** [comment]
@anpingli 
You reviewed the PRs that added these new parameters. Can you review the docs for the parameters?
Thank you!

**Comment by mburke5678** [comment]
QE verified in BZ.

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -209,6 +209,10 @@ when `openshift_logging_use_ops` is set to `true`.
 |Equivalent to `openshift_logging_curator_memory_limit` for Ops cluster
 when `openshift_logging_use_ops` is set to `true`.
 
+|`openshift_logging_curator_replace_configmap`
+|Set to `no` to prevent the upgrade from replacing the `logging-curator` ConfigMap.
+Set to `yes` to allow the ConfigMap to be overriden.
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc:214]: s/overriden/overridden

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -404,6 +408,10 @@ the fields according to the other undefined field settings below.  The default i
 `openshift_logging_fluentd_merge_json_log`. By default, Fluentd removes fields
 with empty values from the record, except for the `message` field.
 
+|`openshift_logging_fluentd_replace_configmap`
+|Set to `no` to prevent the upgrade from replacing the `logging-fluentd` ConfigMap.
+Set to `yes` to allow the ConfigMap to be overriden.
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc:413]: s/overriden/overridden

**Comment by ahardin-rh** [comment]
Just minor comments from me. Otherwise, looks good!

**Pull request merged**